### PR TITLE
1555 link to contributing guide in FAQ

### LINKED ISSUE #1555: Link contributing to the kodadot faq
Link https://github.com/kodadot/nft-gallery/blob/main/CONTRIBUTING.md as lot of devs fails to find this stuff 

With some initial few sentences

### PR DESCRIPTION
**Pull request by kkukelka**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇  _ Do a quick check before the merge. 

### PR type
- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

### Before submitting Pull Request, please make sure:
- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted screenshot of demonstrated change in this PR

Screenshot 2021-12-17 at 14 08 26

### Optional
- [ ] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [ ] I've tested PR on mobile and everything works
- [ ] I found edge cases

### What's new?
- [x] PR closes #1555
- [x] added contributing item in faq section

### Had issue bounty label ?
- [x] Fill up your KSM address: [Payout](https://kodadot.xyz/transfer/?target=EqdyzrzVmeHwMdMwvPeCMnNdbuQDbD3YrjY93xq9Ln3jUGW)

### Community participation
- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot
- [ ] My fix has changed **something** on UI, a screenshot for others, is best to understand changes.

### CODE REVIEW

File: langDir/all_lang.json [hunk 1]
```diff
@@ -14,6 +14,14 @@
         "href": "https://kusama.subscan.io/extrinsic/6095478-1"
       }
     },
+    "kusama_github": {
+      "open_issues": {
+        "href": "https://github.com/kodadot/nft-gallery/issues"
+      },
+      "contributing_guide": {
+        "href": "https://github.com/kodadot/nft-gallery/blob/main/CONTRIBUTING.md"
+      }
```
Comment (roiLeo) [inline langDir/all_lang.json:23]: Hey! you should move this to `kodadot_social: {}`

**Review by roiLeo — CHANGES_REQUESTED** [review]

File: langDir/all_lang.json (hunk 1 above)
Comment (kkukelka) [reply langDir/all_lang.json]: done!

**Review by roiLeo — APPROVED** [review]
lgtm

**Comment by yangwao** [comment]
yas

Perfect, I’ve sent the payout https://kusama.subscan.io/extrinsic/0xd8e4efcb6d5d588ea0db160fbe3493df3163a43ab644e9dc64d5089f55bb16b3

Let’s grab another issue and get rewarded! 
https://github.com/kodadot/nft-gallery/issues/

**Pull request merged**